xbmc/filesystem/CurlFile.cpp

```diff
@@ -667,6 +669,25 @@ void CCurlFile::SetCommonOptions(CReadState* state, bool failOnError /* = true *
     // enable HTTP2 support. default: CURL_HTTP_VERSION_1_1. Curl >= 7.62.0 defaults to CURL_HTTP_VERSION_2TLS
     g_curlInterface.easy_setopt(h, CURLOPT_HTTP_VERSION, CURL_HTTP_VERSION_2TLS);
 
+#if LIBCURL_VERSION_NUM >= 0x074001 // 7.64.1
+  const CURLcode altSvcCacheResult =
+    g_curlInterface.easy_setopt(h, CURLOPT_ALTSVC, m_curlAltSvcCachePath.c_str());
+  if (altSvcCacheResult == CURLE_OK)
+  {
+    CLog::Log(
+      LOGDEBUG,
+      "CCurlFile::SetCommonOptions - alt svc path set to %s",
```

Comment:
Can this spam the debug log if extremely frequent requests are made?

Reply:
I believe so.

However, there are plenty of other places for that to happen too; you'll see many `CLog::Log` invocations in commonly run functions in this file.

Is that a problem? If so, is there a change I can make to fix it?

Reply:
The change is just to drop some logging. You know the here code better I, so the decision is yours. I was just highlighting that it's a decision to consider.